### PR TITLE
bugfix(snipping): features with empty regions return snips filled with NaNs

### PR DESCRIPTION
**Pull request by agalitsyna**
Related to https://github.com/open2c/cooltools/commit/1a637919bc8c1f2bee31a605491fb7d75fb65221
which turned out to be not a complete fix.

### CODE REVIEW

File: cooltools/snipping.py [hunk 1]
```diff
@@ -134,6 +134,18 @@ def assign_regions(features, supports):
 
 def _pileup(data_select, data_snip, arg):
     support, feature_group = arg
+
+    # check if region is annotated
+    if pd.isnull(support):
+        lo = feature_group["lo"].values
+        hi = feature_group["hi"].values
+        s = hi-lo
+        stack = list(map(np.empty, zip(s, s)))
```
Comment (nvictus) [inline cooltools/snipping.py:143]: replace with a 3D `np.full( shape, np.nan )`

**Comment by agalitsyna** [comment]
Replaced with np.full, and some docstrings added. Won't it be good to have tests for snipping?

**Comment by agalitsyna** [comment]
Or it can wait until the new snipping API?

**Comment by agalitsyna** [comment]
Snipping update is ready for merge

**Comment by nvictus** [comment]
Thank you!

**Pull request merged**